### DISCUSSION TITLE
code/line moves

### DISCUSSION THREAD

**Discussion by prgmr-1** [codefori/vscode-ibmi#2184] [IBM i] [answered]
in SEU and RDi you can move likes of code by placing a CC at beginning line number and CC at the ending then an A or B for after or before.  how does this function work in VSCode.  Been using cut/paste with clunky results

**Comment by chrjorgensen** [comment, marked as answer]
@prgmr-1 Please read the [documentation for VS Code](https://code.visualstudio.com/docs) - it is fully described how to cut/copy lines and insert them in other places (hint: Ctrl-X/Ctrl-C and Ctrl-V without any marking). 😉

**Reply by prgmr-1** [reply]
thx
